src/modules/Servicehosting/Service.php

```diff
@@ -691,6 +696,7 @@ public function updateServer(\Model_ServiceHostingServer $model, array $data)
         $model->manager = $data['manager'] ?? $model->manager;
         $model->accesshash = $data['accesshash'] ?? $model->accesshash;
         $model->port = $data['port'] ?? $model->port;
+        $model->config = json_encode($data['config']) ?? $model->config;
```

Comment:
This is fine as long as the `config` array key is always set, otherwise it will produce an "undefined array key" error

Reply:
the $data array is provided from the request of the form located in the twig file. so i think even if the value of `config` is null, the `config` array key will always exist. No ?